go/vt/vttablet/tabletserver/system_health.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tabletserver
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/shirou/gopsutil/cpu"
```

Comment:
I think we want to use `github.com/shirou/gopsutil/v3/cpu` here to use the `v3` API? See https://github.com/shirou/gopsutil#v3-migration